HPXMLtoOpenStudio/resources/defaults.rb

```diff
@@ -2238,21 +2278,42 @@ def self.apply_hvac(runner, hpxml_bldg, weather, convert_shared_systems, unit_nu
       end
     end
 
+    # EER
+    (hpxml_bldg.cooling_systems + hpxml_bldg.heat_pumps).each do |hvac_system|
+      if hvac_system.is_a?(HPXML::CoolingSystem)
+        next unless [HPXML::HVACTypeCentralAirConditioner,
+                     HPXML::HVACTypeMiniSplitAirConditioner].include? hvac_system.cooling_system_type
+      elsif hvac_system.is_a?(HPXML::HeatPump)
+        next unless [HPXML::HVACTypeHeatPumpAirToAir,
+                     HPXML::HVACTypeHeatPumpMiniSplit].include? hvac_system.heat_pump_type
+      end
+      next unless hvac_system.cooling_efficiency_eer.nil? && hvac_system.cooling_efficiency_eer2.nil?
+
+      seer = hvac_system.cooling_efficiency_seer
+      case hvac_system.compressor_type
+      when HPXML::HVACCompressorTypeSingleStage
+        # FIXME: Review compared to AHRI data
+        cop = 0.2692 * seer + 0.2706 # Regression based on inverse model
+      when HPXML::HVACCompressorTypeTwoStage
+        # FIXME: Review compared to AHRI data
+        cop = 0.2773 * seer - 0.0018 # Regression based on inverse model
+      when HPXML::HVACCompressorTypeVariableSpeed
+        # FIXME: Need to develop something here based on AHRI/NEEP data
+        cop = 4.5
+      end
+      hvac_system.cooling_efficiency_eer = UnitConversions.convert(cop, 'W', 'Btu/hr').round(2)
```

Comment:
I will look at AHRI or ENERGY STAR database to derive new defaults for EER2 (as a function of SEER2 and # speeds). Also need to check whether these values are being used as net or gross values.